lib/src/signed_video_tlv.c

```diff
@@ -320,6 +322,9 @@ decode_general(signed_video_t *self, const uint8_t *data, size_t data_size)
         self->latest_validation->timestamp = gop_info->timestamp;
       }
     }
+    size_t hash_size = data_size - (data_ptr - data);
+    memcpy(self->recived_nalu_list_hash, data_ptr, hash_size);
+    data_ptr += hash_size;
```

Comment:
Consider this use case:
Cameras today generate Signed Video using v1.1.29. If the user checks out signed-video-framework from master and builds the validator to validate that video. What will happen when reaching this point in the code?

Reply:
As far as I can see from the code `gop_hash_off` is always `false` when reaching this point. Hence, the if-statement adds nothing and validating an older video will still fail.